### PR TITLE
faq: add some questions and answers from the wiki

### PR DESCRIPTION
**Pull request by matttbe**
Better to move all the questions here on the website, than having some there, and some here.

I adapted some of the answers. You can find the original ones [here](https://github.com/multipath-tcp/mptcp_net-next/wiki/Home/_compare/34ba2883e2a30b15a149df94a941b446eb51e1d6...6a6efb312538dc83534e09a76c1cbd90f7dfaa9a)

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -153,3 +153,48 @@ defined:
 #define IPPROTO_MPTCP 262
 #endif
 ```
+
+## How to check if MPTCP is working?
+
+The easiest way is to check with a special test server than will if MPTCP is
+being used, and will return the info to the client, e.g.
```
Comment (mjmartineau) [inline faq.md:160]:
```suggestion
The easiest way is to check with a special test server that will confirm MPTCP is
being used and return that info to the client, e.g.
```

File: faq.md [hunk 2]
```diff
@@ -153,3 +153,48 @@ defined:
...
+    --pull always mptcp/mptcp-upstream-virtme-docker:latest \
+    manual-normal
+```
+More details on the
+(MPTCP Upstream Virtme Docker)[https://github.com/multipath-tcp/mptcp-upstream-virtme-docker#readme)
+page.
+
+## High number of retransmissions / dropped packets at the NIX RX queue level?
+
+Even if with MPTCP, the subflow processing is done by the TCP stack, the main
```
Comment (mjmartineau) [inline faq.md:188]:
```suggestion
Even with MPTCP, subflow processing is done by the TCP stack. The main
```

File: faq.md [hunk 3]
```diff
@@ -153,3 +153,48 @@ defined:
...
+difference with plain TCP is that this processing does not use the socket
+backlog and always happens in second-level interrupt handlers called
+[*bottom half*](https://en.wikipedia.org/wiki/Interrupt_handler) (BH) in Linux.
+When the host is under heavy load, BH processing happens in `ksoftirqd` context,
+and there is some latency between the `ksoftirqd` scheduling and the moment
+`ksoftirqd` actually runs that. This depends on the process scheduler decisions
```
Comment (mjmartineau) [inline faq.md:194]:
```suggestion
`ksoftirqd` actually runs the handler. This depends on process scheduler decisions
```

File: faq.md [hunk 4]
```diff
@@ -153,3 +153,48 @@ defined:
...
+(and settings).
+
+A way to reduce these retransmissions and avoid the dropped packets at the NIC
```
Comment (mjmartineau) [inline faq.md:197]:
```suggestion
A way to reduce these retransmissions and avoid dropped packets at the NIC
```

**Review by mjmartineau — COMMENTED** [review]
A few minor adjustments, otherwise looks good! Thanks for moving the FAQ to the main web site.

**Comment by matttbe** [comment]
> A few minor adjustments, otherwise looks good!

Thank you for the review, much appreciated!

**Pull request merged**